src/mdns/mdns_avahi.cpp

```diff
@@ -393,7 +393,6 @@ void PublisherAvahi::HandleGroupState(AvahiEntryGroup *aGroup, AvahiEntryGroupSt
 
     case AVAHI_ENTRY_GROUP_UNCOMMITED:
     case AVAHI_ENTRY_GROUP_REGISTERING:
-        otbrLogErr("Group ready.");
```

Comment:
Why deleting it instead of using a lower log level?

Reply:
There are significant number of this logs. I guess the real question is "Is the log useful in anyway?".

Reply:
Then I think it's fine to remove it.